brd-android/app/src/main/java/com/breadwallet/app/BreadApp.kt

```diff
@@ -145,18 +145,20 @@ class BreadApp : Application(), KodeinAware, CameraXConfig.Provider {
             )
             else -> listOf(
                 "bitcoin-mainnet:__native__",
-                "ethereum-mainnet:__native__"
+                "ethereum-mainnet:__native__",
+                "bitcoinsv-mainnet:__native__"
```

Comment:
could we make sure BSV is on the top?
Victor said to have an  alphabetical order.